README.md

```diff
@@ -38,11 +38,13 @@ Search API provides access to this metadata.
 
 ## Prerequisites
 
-- Java 1.8.0 or higher
+- Java 1.8.0 (a.k.a. JAVA8) only; higher versions will fail.
```

Comment:
Instead of "higher versions will fail" let's say "higher versions are not currently supported" ...